app/pages/nonsipp/membertransferout/ReceivingSchemeNamePage.scala

```diff
@@ -29,3 +29,10 @@ case class ReceivingSchemeNamePage(srn: Srn, index: Max300, transferIndex: Max5)
 
   override def toString: String = "schemeName"
 }
+
+case class ReceivingSchemeNamePages(srn: Srn, index: Max300) extends QuestionPage[Map[String, String]] {
+  override def path: JsPath =
+    Paths.properties \ toString \ index.arrayIndex.toString
```

Comment:
I appreciate this is was done earlier and not part of the ticket @narendrapaduchuri , but this supposed to use the path defined in package.scala to save the data to the right path.